### PR TITLE
Changes to cookbook.md

### LINKED ISSUE #1274: Cookbook code is not Python3
Title says it all.

### PR DESCRIPTION
**Pull request by gedeck**
Changes:

- Modify all print statements to work with Python 2 and 3 (fixes #1274)
- Remove deprecation warning from scikit-learn (fixes #1273)

### CODE REVIEW

File: Docs/Book/Cookbook.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ Examples of using it:
     rms = rdMolAlign.AlignMol(mol1, mol2)
     tani = rdShapeHelpers.ShapeTanimotoDist(mol1, mol2)
     prtr = rdShapeHelpers.ShapeProtrudeDist(mol1, mol2)
-    print rms, tani, prtr
+    print('{0:.11f} {1:.11f} {2:.13f}'.format(rms, tani, prtr))
```
Comment (gedeck) [inline Docs/Book/Cookbook.md]: This looks overkill, but will reproduce the expected result. We could change this to a more reasonable format statement.

File: Docs/Book/Cookbook.md [hunk 2]
```diff
@@ -291,7 +291,7 @@ Examples of using it:
     fp2 = Generate.Gen2DFingerprint(mol2, factory, dMat=Chem.Get3DDistanceMatrix(mol2))
     # Tanimoto similarity
     tani = DataStructs.TanimotoSimilarity(fp1, fp2)
-    print tani
+    print('{0:.12f}'.format(tani))
```
Comment (gedeck) [inline Docs/Book/Cookbook.md]: as above

File: Docs/Book/Cookbook.md [hunk 3]
```diff
@@ -323,7 +323,7 @@ Examples of using it:
     tfd1 = TorsionFingerprints.GetTFDBetweenMolecules(mol1, mol2)
     tfd2 = TorsionFingerprints.GetTFDBetweenMolecules(mol1, mol2, useWeights=False)
     tfd3 = TorsionFingerprints.GetTFDBetweenMolecules(mol1, mol2, maxDev='spec')
-    print tfd1, tfd2, tfd3
+    print('{:.13f} {:.12f} {:.13f}'.format(tfd1, tfd2, tfd3))
```
Comment (gedeck) [inline Docs/Book/Cookbook.md]: and again

File: Docs/Book/Cookbook.md [hunk 4]
```diff
@@ -409,8 +409,8 @@ The code:
     fp = numpy.zeros((1,))
     DataStructs.ConvertToNumpyArray(AllChem.GetMorganFingerprintAsBitVect(m5, 2), fp)
 
-    print rf.predict(fp)
-    print rf.predict_proba(fp)
+    print(rf.predict((fp,)))
+    print(rf.predict_proba((fp,)))
```
Comment (gedeck) [inline Docs/Book/Cookbook.md]: That is the fix for the deprecation warning

File: Docs/Book/Cookbook.md [hunk 5]
```diff
@@ -425,7 +425,7 @@ The code:
 
     # helper function
     def getProba(fp, predictionFunction):
-      return predictionFunction(fp)[0][1]
+      return predictionFunction((fp,))[0][1]
```
Comment (gedeck) [inline Docs/Book/Cookbook.md]: as above

File: Docs/Book/Cookbook.md (hunk 1 above)
Comment (greglandrum) [inline Docs/Book/Cookbook.md]: I appreciate the effort, but I think it's distracting. We aren't testing the cookbook, so this degree of precision isn't necessary.
How about just doing a normal print here (and elsewhere)?

**Pull request merged**